utils/codepointtrie/src/codepointtrie.rs

```diff
@@ -390,6 +395,7 @@ impl<'trie, T: TrieValue> CodePointTrie<'trie, T> {
             header: self.header,
             index: self.index,
             data: converted_data,
+            error_value: self.error_value,
```

Comment:
hm, @sffc how should try_into_converted work here? should the error value also just be casted with a try_into?

Reply:
The way to do this with ZeroVec would be:

 - Convert `self.error_value` to `T::ULE` by calling `self.error_value.to_unaligned()`
 - Construct a `ZeroVec::Borrowed(&[ule_value])`
 - Call `try_into_converted()` on _that_
 - Call `.get(0)` on that and use that value

Reply:
Hmm. Hadn't considered that. @Manishearth's workaround sounds good to me.